dist/openshift/cincinnati.yaml

```diff
@@ -266,7 +266,7 @@ parameters:
     displayName: "Policy-engine CPU request"
     description: "Requested amount of CPU (millicores) allowed for policy-engine (default: 350m)"
   - name: GB_PAUSE_SECS
-    value: "30"
+    value: "600"
```

Comment:
10 minutes is a very big change from 30 seconds. I am fine with anything less than 5 mins as mentioned in https://github.com/openshift/cincinnati/pull/264#issuecomment-608335262

Reply:
Shifted to 5m with 000eacc -> 3449492.